.drone.yml

```diff
@@ -33,7 +33,7 @@ pipeline:
       - rake push
     when:
       event: tag
-      branch: master
+      # branch: master
```

Comment:
How does this work? Won't it need to be merged in to work?

Reply:
current master represents `2.x` version with new aws provider which is required for vault. In this PR, I am trying to update the amis and release the `terrafying-components` for all other systems except vault which are using `1.x` version (with old aws provider).

In order to do that, we need to maintain 2 versions of `terrafying-components`, master branch which points to `2.x` and this branch which will point to `1.x`

tag and gem gets published only if the branch to master, but we don't want it to merge to master and maintain separate branch. hence I am removing branch condition from drone step. In this way, I will be able to release the tag and release the gem even from the non master branch.

Reply:
Yeah I understand that part. What I mean is, does the drone part need to be merged in in order for a tag on a non-master branch to trigger this step?

Reply:
sure I will try and see sample tag and see how it goes :+1:

Reply:
Cool, so just for my clarification, the drone.yml can be changed in the branch and any added/removed steps will be used in drone